.gitignore

```diff
@@ -1,34 +1,17 @@
 # Xcode
 #
-build/
-*.pbxuser
-!default.pbxuser
-*.mode1v3
-!default.mode1v3
-*.mode2v3
-!default.mode2v3
-*.perspectivev3
-!default.perspectivev3
-project.xcworkspace
-!default.xcworkspace
-xcuserdata
-*.xccheckout
-*.moved-aside
-DerivedData
-*.hmap
-*.ipa
-*.xcuserstate
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
+xcuserdata/
 
 # Carthage
 Carthage/Build
 
-# CocoaPods
-#
-# We recommend against adding the Pods directory to your .gitignore. However
-# you should judge for yourself, the pros and cons are mentioned at:
-# http://guides.cocoapods.org/using/using-cocoapods.html#should-i-ignore-the-pods-directory-in-source-control
-#
-Pods/
-Podfile.lock
+# Lumberjack generated files
 LumberjackUser*.h
-.DS_Store
+
+# SwiftPM
+.swiftpm
+.build
```

Comment:
`/.build` is already excluded above.
We should also only exclude `.swiftpm` on the root level (as in `/.swiftpm`) and move it up to the `/.build` definition.